controllers/tables.go

```
@@ -22,13 +22,7 @@ func GetTables(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	order, err := postgres.OrderByRequest(r)
```

Never ignore errors! Please check all ignored errors